src/main/java/roomescape/controller/MemberController.java

```diff
@@ -0,0 +1,59 @@
+package roomescape.controller;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.model.Member;
+
+@Controller
+public class MemberController {
+    private final List<Member> members = new ArrayList<>();
+    private final AtomicLong index = new AtomicLong(1);
+    private static final Logger log = LoggerFactory.getLogger(MemberController.class);
+
+    @GetMapping("/reservation")
+    public String reservation() {
+        return "reservation";
+    }
+
+    @GetMapping("/reservations")
+    public ResponseEntity<List<Member>> read() {
+        return ResponseEntity.ok(members);
+    }
+
+    @PostMapping("/reservations")
+    public ResponseEntity<Member> create(@RequestBody Member member) {
+        if(member.getName() == null || member.getName().isEmpty()) {
+            throw new IllegalArgumentException("name 은 필수 값입니다.");
+        }
+        if(member.getDate() == null || member.getDate().isEmpty()) {
+            throw new IllegalArgumentException("date 는 필수 값입니다.");
+        }
+        if(member.getTime() == null || member.getTime().isEmpty()) {
+            throw new IllegalArgumentException("time 은 필수 값입니다.");
+        }
```

Comment:
null check를 꼼꼼하게 잘 해주셨네요!  👍

> 참고로 인텔리제이에서는 Cmd + Opt + L 로 라인 포맷팅을 할 수 있습니다~!

```suggestion
        if (member.getName() == null || member.getName().isEmpty()) {
            throw new IllegalArgumentException("name 은 필수 값입니다.");
        }
        if (member.getDate() == null || member.getDate().isEmpty()) {
            throw new IllegalArgumentException("date 는 필수 값입니다.");
        }
        if (member.getTime() == null || member.getTime().isEmpty()) {
            throw new IllegalArgumentException("time 은 필수 값입니다.");
        }
```

Reply:
> 참고로 인텔리제이에서는 Cmd + Opt + L 로 라인 포맷팅을 할 수 있습니다~!

헉 찾고 있던 꿀팁이었습니다~! 🤤